app/crawler/naver_map_crawler.py

```
@@ -147,35 +160,100 @@ def _search_sync(self, query: str) -> List[Dict[str, str]]:
                 
         return list(results.values())
 
+    def _wait_with_jitter(self, base_ms: int) -> int:
+        return max(0, int(base_ms + random.randint(0, self.PAGE_WAIT_JITTER_MS)))
```

Ruff (0.15.2)

[error] Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)